openpype/plugins/publish/integrate.py

```diff
@@ -137,7 +137,8 @@ class IntegrateAsset(pyblish.api.InstancePlugin):
                 "mvUsdOverride",
                 "simpleUnrealTexture",
                 "online",
-                "uasset"
+                "uasset",
+                "bgeo"
```

Comment:
Technically `bgeo` could be removed again right, since we now still rely on the `pointcache` family existing? If we expect `bgeo` to remain here, shouldn't we now also add the new `abc` family?

Reply:
good catch! 👍🏻